src/runtime_src/core/tools/common/ReportCu.cpp

```diff
@@ -112,23 +112,27 @@ populate_cus_new(const xrt_core::device *device)
     return pt;
   }
 
+  boost::property_tree::ptree cu_list;
   for (auto& stat : cu_stats) {
     boost::property_tree::ptree ptCu;
     ptCu.put( "name",           stat.name);
     ptCu.put( "base_address",   boost::str(boost::format("0x%x") % stat.base_addr));
     ptCu.put( "usage",          stat.usages);
     ptCu.add_child( std::string("status"),	get_cu_status(stat.status));
-    pt.push_back(std::make_pair("", ptCu));
+    cu_list.push_back(std::make_pair("", ptCu));
   }
+  pt.add_child("compute_units", cu_list);

```

Comment:
Can we named it "pl_compute_units".
Xilinx FPGA user would use terminology "PL" and "PS".
Where PL means Programmable Logic, PS means Processing System.

In this change, we already have "ps_compute_units" for PS kernels. :)

Reply:
Yes, I just updated it. Thanks for the suggestion :)